### DISCUSSION TITLE
Support reStructuredText (RST) in READMEs

### DISCUSSION THREAD

**Discussion by kdewald** [rust-lang/crates.io#6106] [Feature Requests]
Currently crates.io only supports markdown syntax in README files. Other repositories and registries also usually support RST as an additional format, mainly to provide more flexibility on what can be rendered.

Would it be possible to add something like this? (I'm also more than happy to try it if someone can guide me a bit in the right direction)

Thanks!

**Comment by Turbo87** [comment]
While it would technically be possible to support rST, it would increase the complexity of our Readme rendering code quite a bit and it is already not one of the most straight forward parts of the codebase. Since rustdoc only supports markdown too I'm inclined to say that we currently wouldn't want to add support for rST either. Sorry...! 😐
